bc_obps/service/operation_service.py

```diff
@@ -545,6 +545,24 @@ def check_conditions() -> Generator[Tuple[Callable[[], bool], str], None, None]:
                 ),
                 "Operation must have a process flow diagram and a boundary map.",
             )
+            # Check if operation documents have been scanned for malware
+            yield (
+                lambda: not (
+                    operation.registration_purpose != Operation.Purposes.ELECTRICITY_IMPORT_OPERATION
```

Comment:
Does this mean EIOs can have unscanned documents? Could we just remove this condition?

just a simple
```py
yield 
    lambda: not operation.documents.filter(
          status=Document.FileStatus.UNSCANNED,
        ).exists()
```

Reply:
Fixed with [1b29670](https://github.com/bcgov/cas-registration/pull/3137/commits/1b29670107d55f00234ed68a42bc1bf4cb084f2b)